websites/P/Peario/dist/metadata.json

```diff
@@ -0,0 +1,44 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Dark_Ville",
+		"id": "638080361179512853"
+	},
+	"service": "Peario",
+	"description": {
+		"nl": "Deel je streams. Peario zorgt er voor dat je Stremio streams kan delen met je vrienden.",
+		"en": "Share your streams. Peario allows you to watch Stremio streams with your friends."
+	},
+	"url": [
+		"peario.xyz",
+		"www.peario.xyz"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/mtqwu0r.png",
```

Comment:
Not best but better
https://i.imgur.com/7wb2oQa.png

Reply:
I’ll put one through waifu2x and will change it.